batch/src/main/kotlin/org/yapp/batch/config/FcmConfig.kt

```
@@ -0,0 +1,87 @@
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.io.ByteArrayInputStream
+import java.io.IOException
+import java.util.Base64
+
+@Configuration
+class FcmConfig {
+
+    @Value("\${FIREBASE_TYPE:service_account}")
+    private lateinit var type: String
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)